components/Dropdowns/DropdownRatings.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from "classnames";
+import Image from "next/image";
+import React from "react";
+
+interface DropdownRatings {
+  max: number;
+  close: () => void;
+  value?: number;
+  onChange: (field: number) => void;
+  onFocus?: () => void;
+  closeOnSelect?: boolean;
+  borders?: boolean;
+}
+
+export const DropdownRatings: React.FC<DropdownRatings> = ({
+  max,
+  close,
+  value,
+  onChange,
+  onFocus,
+  closeOnSelect = false,
+  borders = false,
+}) => {
+  React.useEffect(() => {
+    return () => {
+      if (onFocus) onFocus();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div
+      className={classNames("grid grid-cols-1 w-full overflow-y-scroll", {
```

Comment:
overflow-y-scroll here seems to be unnecessary. In my browsers (chrome, firefox) it displayed an extra white line. 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/19307482/177387361-98c56a36-3eb8-431b-8068-c3d2f159f93f.png">

Reply:
Would merge after this small fix